lib/3scale/backend/configuration.rb

```diff
@@ -2,7 +2,7 @@
 require '3scale/backend/environment'
 require '3scale/backend/configurable'
 require '3scale/backend/errors'
-require 'dotenv/load' if ENV['RACK_ENV'] == 'development'
+require 'dotenv/load' if ENV['RACK_ENV'] != 'production'
```

Comment:
so we want it in `test`? Just a question in case that was missed.

Reply:
That's fixed in https://github.com/3scale/apisonator/pull/370